great_expectations/expectations/core/expect_column_chisquare_test_p_value_to_be_greater_than.py

```diff
@@ -27,6 +27,12 @@ class ExpectColumnChiSquareTestPValueToBeGreaterThan(TableExpectation):
     metric_dependencies = tuple()
     success_keys = ()
     default_kwarg_values = {}
+    args_keys = (
+        "column",
+        "partition_object",
+        "p",
+        "tail_weight_holdout",
+    )
```

Comment:
Have we documented this general structure anywhere? If all Expectations need success_keys, default_kwarg_values, arg_keys, etc, I'd think we'd want to fulfill some kind of interface right?

I don't want to go against the grain at all if this is how Expectations currently exist but I do want to make sure users are aware that all these components are necessary.

Reply:
This is a great point. I wonder whether this should be in Expectation contribution docs, in another part of our docs, in a docstring in our codebase, or all of the above.

Reply:
@austiezr to take action here to add to the expectation contribution guide.